98-VisualTools/README.md

```diff
@@ -1,9 +1,9 @@
-# GIT Visual Tools
-
-## GitHub
-
-## SourceTree
-
-## TortoiseGIT
-
-## Visual Studio 2013+
+# GIT Visual Tools
+
+## GitHub (free)

```

Comment:
It might be informative to highlight these tools are free.  Feel free to remove these notes.